zsh/PKGBUILD

```diff
@@ -13,6 +13,7 @@ source=("https://www.zsh.org/pub/zsh-${pkgver}.tar.xz"{,.asc}
         'config.guess::https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'
         'config.sub::https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'
         'zprofile'
+        'zshrc'
```

Comment:
bump pkgrel

Reply:
@lazka 

Thank you for your reviewing.
I did it. Please confirm.